openapi/api.yml

```diff
@@ -0,0 +1,82 @@
+openapi: "3.0.0"
+
+info:
+  description: Account store API for DLUHC Funding Service Design
+  version: "0.1.0"
+  title: Funding Service Design - Account store
+paths:
+  /account:
+    get:
+      summary: "Return the account data given the account email address or account id."
+      description: "Given a GET request with the account id or email address given as a query string we return a reponse containing the data associated with the account."
+      operationId: core.account.get_account
+      parameters:
+        - name: email_address
+          in: query
+          description: "The email being used for the account id look up, if both account_id and email_address is given then the look up will use the account_id instead"
+          required: false
+          schema:
+            type: string
+        - name: account_id
+          in: query
+          description: "The account id being used for the account id look up, if both account_id and email_address is given then the look up will use the account_id instead."
+          required: false
+          schema:
+            type: string
+      responses:
+        200:
+          description: The requested account exists, and its json and sent back as a response.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/account'
+        204:
+          description: "The requested account doesn't exist."
+    post:
+      summary: "Post a new email - creating a json in the account store with a unique account id."
```

Comment:
probably should not refer "json in account store", its a level of detail that I dont think helps someone reading the docs. They dont care how you store it, just that they can put data in and get data out

Reply:
Agreed - I like low cognitive load docs.